docs/Searching/Search.md

```diff
@@ -367,6 +367,49 @@ var terms = getInstance( "HyperClient@cbElasticsearch" )
             } );
 ```
 
+## Term Vectors
+
+The ["Term Vectors" Elasticsearch API](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-termvectors.html) allows you to retrieve information and statistics for terms in a specific document field. This could be useful for finding the most common term in a book description, or retrieving all terms with a minimum word length from the book title.
+
+### Retrieving Term Vectors By Document ID
+
+To retrieve term vectors for a known document ID:
+
+```js
+var result = variables.model.getTermVectors(
+    "books",
+    "book_12345",
+    { "fields" : "title" }
+);
+```
+
+Use the third argument, `params`, to configure the request using the [documented query parameters](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-termvectors.html#docs-termvectors-api-query-params): 
+
+```js
+var result = variables.model.getTermVectors(
+    indexName = "books",
+    id = "book_12345",
+    params = {
+        "fields" : "title",
+        "min_word_length" : 4
+    }
+);
+```
+
+### Retrieving Term Vectors By Payload
+
+If you wish to analyze a payload (not an existing document) you can pass a payload in the `body` argument's `"doc"` field:
+
+```js
+var result = variables.model.getTermVectors(
```

Comment:
Let’s change all of these examples to make it clear the methods are operating on the Client.  “variables.model” seems ambiguous.